kornia/enhance/adjust.py

```diff
@@ -406,47 +409,72 @@ def sharpness(input: torch.Tensor, factor: Union[float, torch.Tensor]) -> torch.
 
     Returns:
         torch.Tensor: Sharpened image or images.
+
+    Example:
+        >>> _ = torch.manual_seed(0)
+        >>> sharpness(torch.randn(1, 1, 5, 5), 0.5)
+        tensor([[[[-1.1258, -1.1524, -0.2506, -0.4339,  0.8487],
+                  [ 0.6920, -0.1580, -1.0576,  0.1765, -0.1577],
+                  [ 1.4437,  0.1998,  0.1799,  0.6588, -0.1435],
+                  [-0.1116, -0.3068,  0.8381,  1.3477,  0.0537],
+                  [ 0.6181, -0.4128, -0.8411, -2.3160, -0.1023]]]])
     """
     input = _to_bchw(input)
```

Comment:
this is not related to this PR but shouldn't we accept only bchw to make it seamless to the rest of the library?

Reply:
Yes. It makes sense. I will send another PR for this.